src/views/home.js

```diff
@@ -1,5 +1,20 @@
 import React from 'react'
 import ampersandMixin from 'ampersand-react-mixin'
+import Icon from'react-fa'
+
+const RepoItem = React.createClass({
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).